### PR TITLE
Fix cleared color on shadows

### PR DESCRIPTION
**Pull request by IMS212**
This fixes SEUS PTGI's raytracing, provided you do not use sodium.

### CODE REVIEW

**Comment by ColdIce1605** [comment]
My only question is this default behavior in OF, or shadermod because if it's not maybe this shouldn't be merged and the config option should be worked on which will take time of course.

**Comment by JONTTM** [comment]
> My only question is this default behavior in OF, or shadermod because if it's not maybe this shouldn't be merged and the config option should be worked on which will take time of course.

SEUS PTGI works in OF by default

**Comment by ColdIce1605** [comment]
No not what I meant, what I meant is this clear color the default in OF or does PTGI change it.

File: src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java [hunk 1]
```diff
@@ -292,7 +292,7 @@ public void renderShadows(WorldRendererAccessor worldRenderer, Camera playerCame
 
 		// TODO: Support shadow clear color directives & disable buffer clearing
 		// Ensure that the color and depth values are cleared appropriately
-		RenderSystem.clearColor(0.0f, 0.0f, 0.0f, 0.0f);
+		RenderSystem.clearColor(255, 255, 255, 1);
```
Comment (coderbot16) [inline src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java:295]: Color values should be in the range of 0.0F to 1.0F when calling clearColor, not 0 to 255.

**Review by coderbot16 — CHANGES_REQUESTED** [review]
The call to clearColor is slightly wrong. But I can confirm that the shadow color should be cleared to white, unless otherwise specified by the shader pack.

File: src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java (hunk 1 above)
Comment (IMS212) [reply src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java]: Fixed.

**Pull request closed without merging**